js/gated-content/src/components/blog/BlogListing.vue

```diff
@@ -67,6 +68,16 @@ export default {
       type: Boolean,
       default: false,
     },
+    favorites: {
```

Comment:
I suppose we can move this to a mixin also

Reply:
Fixed